chats/apps/sectorqueue/models.py

```diff
@@ -0,0 +1,100 @@
+from django.db import models
+from chats.core.models import BaseModel
+from django.utils.translation import gettext_lazy as _
+from chats.apps.sectors.models import Sector, SectorAuthorization
+
+
+class SectorQueue(BaseModel):
+    sector = models.ForeignKey(
+        Sector,
+        verbose_name=_("queues"),
+        related_name="sector_authorizations",
+        on_delete=models.CASCADE,
+        to_field="uuid"
+    )
+    name = models.CharField(_("sector name"), max_length=150, blank=True)
+
+    class Meta:
+        verbose_name = _("Sector Queue")
+        verbose_name_plural = _("Sector Queues")
+
+    def __str__(self):
+        return self.name
+
+    def get_permission(self, user):
```

Comment:
Comment this piece of code, explaining whats happening. As it will, possibly, be hard to understand in future usage.

I think you could use [Strategy](https://refactoring.guru/design-patterns/strategy) design pattern to make this more readable.

Reply:
I use comment to explain better how this code works.